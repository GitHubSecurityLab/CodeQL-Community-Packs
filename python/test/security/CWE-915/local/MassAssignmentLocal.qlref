security/CWE-915/MassAssignmentLocalProblem.ql
