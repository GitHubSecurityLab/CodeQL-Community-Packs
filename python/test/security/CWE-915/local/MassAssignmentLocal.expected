| massassignmentLocal.py:18:26:18:28 | ControlFlowNode for key | Use of $@. | massassignmentLocal.py:8:13:8:31 | ControlFlowNode for Attribute() | mass assignment |
| massassignmentLocal.py:18:26:18:28 | ControlFlowNode for key | Use of $@. | massassignmentLocal.py:30:10:30:20 | ControlFlowNode for Attribute | mass assignment |
| massassignmentLocal.py:24:15:24:25 | ControlFlowNode for Attribute | Use of $@. | massassignmentLocal.py:8:13:8:31 | ControlFlowNode for Attribute() | mass assignment |
| massassignmentLocal.py:24:15:24:25 | ControlFlowNode for Attribute | Use of $@. | massassignmentLocal.py:24:15:24:25 | ControlFlowNode for Attribute | mass assignment |
| massassignmentLocal.py:27:18:27:28 | ControlFlowNode for Attribute | Use of $@. | massassignmentLocal.py:8:13:8:31 | ControlFlowNode for Attribute() | mass assignment |
| massassignmentLocal.py:27:18:27:28 | ControlFlowNode for Attribute | Use of $@. | massassignmentLocal.py:27:18:27:28 | ControlFlowNode for Attribute | mass assignment |
